modules/qna/src/backend/transfer.ts

```diff
@@ -1,9 +1,52 @@
-import { QnaItem, QnaEntry } from './qna'
+import * as sdk from 'botpress/sdk'
+import { validate } from 'joi'
+import _ from 'lodash'
+
+import { QnaEntry, QnaItem } from './qna'
 import Storage from './storage'
+import { QnaItemArraySchema, QnaItemCmsArraySchema } from './validation'
+
+const debug = DEBUG('qna:import')
+
+type ContentData = Pick<sdk.ContentElement, 'id' | 'contentType' | 'formData'>
+
+interface ImportData {
+  questions?: QnaItem[]
+  content?: ContentData[]
+}
+
+export const prepareImport = async (parsedJson: any): Promise<ImportData> => {
+  try {
```

Comment:
I understand that you do this double "Validate" just to be backward compatible am I right? If yes can you add a deprecated comment just to make sure we get rid of this in bp next

Reply:
Yup, but i'm not sure it would be a good idea to deprecate that. Import / export should always be backward compatible to make it easier for someone to migrate an old bot to a new version